SetDefaultSinkId/explainer.md

```diff
@@ -0,0 +1,185 @@
+# Set Default Audio Output Device
+
+Author: [Sunggook Chue] [Ravikiran Ramachandra] [Steve Becker] [Andy Luhrs]
+
+## Status of this Document
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+
+This document status: Active
+Expected venue: [WebRTC WG](https://www.w3.org/groups/wg/webrtc)
+Current version: this document
+
+## Introduction
+
+Setting Audio Output Device In Frames
+
+In embedded web experiences, the top-level frame grants iframes access to system media devices like microphones, cameras and speakers. This is typically done through the permission policy (https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) in the iframe tag. However, this access comes with two key challenges:
+
+- Independent Choice: Each iframe independently chooses its own media device. The top-level frame cannot directly influence or view this selection due to browser security restrictions (cross-origin boundary).
```

Comment:
Nit: lower case second word in this and the next bullet; they're not proper names.